wg-deployment/charter.md

```diff
@@ -0,0 +1,51 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
+Develop and maintain tools like kfctl (CLI + operator) for using the manifests to deploy Kubeflow.
+
+### In scope
+
+#### Responsibilities
+
+##### Application Owners
+
+- Ensure application manifests are up to date with the latest changes.
+
+##### Platform Owners
+
+- Ensure platform-specific deployment artifacts (manifests, KFDefs, kpt functions) are up-to-date.
+
+##### Deployment Working Group Owners
+
+- Work with Application Owners and Platform Owners to make the integration and release process easier.
+
+#### Code, Binaries and Services
+
+- kfctl
+- kfdef for each platform
+- Testing
+- Infrastructure for testing
+
+#### Cross-cutting and Externally Facing Processes
+
+- Cutting releases on both the manifests and kfctl repos
+- Qualifying a Kubeflow release for each platform
+- Co-ordinating with application OWNERs before a release to qualify application releases
+
+### Out of scope
+
```

Comment:
Do you want to call out maintaining manifests for individual applications is out of scope for this WG? That this should be the responsibility of the application OWNERs and WGs?

Reply:
Yeah, I've listed it as an responsibility of the application OWNERs. I'll add that in out of scope for this WG